doc/source/user_guide/meshing_workflow/meshing_workflows.rst

```diff
@@ -572,6 +572,220 @@ Switch to solution mode
 
     solver = meshing.switch_to_solver()
 
+
+2D meshing workflow
+-------------------
+Use the **2D** meshing workflow to mesh specific two dimensional geometries.
+The following example shows you how to use the 2D Meshing workflow.
+
+Import geometry
+~~~~~~~~~~~~~~~
+
+.. code:: python
+
+    import ansys.fluent.core as pyfluent
+    from ansys.fluent.core import examples
+
+    import_file_name = examples.download_file('NACA0012.fmd', 'pyfluent/airfoils')
+    meshing = pyfluent.launch_fluent(
+        mode="meshing", precision='double', processor_count=2
+    )
+    meshing.workflow.InitializeWorkflow(WorkflowType="2D Meshing")
+    meshing.workflow.TaskObject["Load CAD Geometry"].Arguments.set_state(
+        {
+            r"FileName": import_file_name,
+            r"LengthUnit": r"mm",
+            r"Refaceting": {
+                r"Refacet": False,
+            },
+        }
+    )
+    meshing.workflow.TaskObject["Load CAD Geometry"].Execute()
+
+Set regions and boundaries
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. code:: python
+
+    meshing.workflow.TaskObject["Update Regions"].Execute()
+    meshing.workflow.TaskObject["Update Boundaries"].Arguments.set_state(
+        {
+            r"SelectionType": r"zone",
+        }
+    )
+    meshing.workflow.TaskObject["Update Boundaries"].Execute()
+
+Define global sizing
+~~~~~~~~~~~~~~~~~~~~
+
+.. code:: python
+
+    meshing.workflow.TaskObject["Define Global Sizing"].Arguments.set_state(
+        {
+            r"CurvatureNormalAngle": 20,
+            r"MaxSize": 2000,
+            r"MinSize": 5,
+            r"SizeFunctions": r"Curvature",
+        }
+    )
+    meshing.workflow.TaskObject["Define Global Sizing"].Execute()
+
+Adding BOI
+~~~~~~~~~~
```

Comment:
```suggestion
Add body of influence
~~~~~~~~~~~~~~
```
I don't think we can assume someone will know what BOI means, especially in a title. The code itself implicitly defines it. However, a glance at the heading would likely be confusing.